packages/ui-tests/src/Menu.tsx

```diff
@@ -1,14 +1,20 @@
 import React, { useCallback, useState, useMemo } from 'react'
 import styled, { css } from 'styled-components'
-import { useHistory } from 'react-router-dom'
+import { useNavigate, useLocation } from 'react-router-dom'
 import {
   ExpandableList,
   ExpandableListItemType,
   IconButton,
 } from 'practical-react-components-core'
 import { HamburgerMenuIcon } from 'practical-react-components-icons'
 
-import { Components, Component } from './types'
+interface Component {
+  readonly name: string
+  readonly route: string
+  readonly menu: string
+  readonly component: React.FC
+}
+type Components = ReadonlyArray<Component>
```

Comment:
Why not import these anymore?

Reply:
They were never imported, I got error in VS code direct. Seems like the linting is wrong in this repository and tsc is not run through all the files in this repository.  These components exist only in `packages/docs/src/...` and IMO it's not a good idea to import them from another package in a monorepo.